resources/assets/js/components/people/ContactInformation.vue

```diff
@@ -35,7 +35,7 @@
         </div>
 
         <div class="w-100" v-show="contactInformation.edit">
-          <form class="measure center">
+          <form class="measure center" v-on:submit.prevent>
```

Comment:
We could allow user to hit `Enter`, and just update this form
```suggestion
          <form class="measure center" v-on:submit.prevent="update(contactInformation)">
```